ggml/src/ggml-metal/ggml-metal-impl.h

```diff
@@ -285,4 +285,22 @@ typedef struct {
     float    eps;
 } ggml_metal_kargs_rms_norm;
 
+typedef struct {
+    int32_t  ofs0;
+    int32_t  ofs1;
```

Comment:
These have to be `uint64_t` since they are memory address strides. Also change the types in `ggml_metal.m` on lines 3281,3282 to `uint64_t`. Ideally there should be no division by `4` there - it assumes the data is `float`.

You can also just leave a comment TODO here and I'll adjust these later.

Reply:
Ok, I have changed the type.